src/components/SignUp/index.js

```diff
@@ -0,0 +1,261 @@
+import {
+  Button,
+  Typography,
+  Grid,
+  TextField,
+  InputAdornment,
+} from "@material-ui/core";
+import { useFormik } from "formik";
```

Comment:
Haven't played with `Formik` since nigerianstat but I kind of remember `Formik` mentioning `useFormik` should be used very [rarely](https://formik.org/docs/api/useFormik). Why are we using now?

Reply:
Would be great if you also linked something explaining to why it was mentioned , that would be helpful :) I used this and I wasn't aware of that, It was listed as a way to help with validation, which was the problem I was trying to solve at the moment @kilemensi

Reply:
oh you did link lol, let me have a look

Reply:
Just had a read and it mentioned the use cases are rare and to only use it when you are not using `withFormik` or `Formik`, which we are not so I believe it matches our use case, plus the example given on the docs is exactly how we have used it here @kilemensi

Reply:
What I understood from the doc @gertie-sheshe is: Most of the time, you should use `Formik` or `withFormik`. The 3 cases they (give in that doc) for when one should prefer `useFormik` are:
 
 * You are Jared,
 * You are modifying the returned value and creating a modified version of <Formik> for your own consumption, and
 * You want to avoid using React Context (possibly for perf reasons).
 
 Which use case is applicable to us( and why)?

Reply:
- You are Jared 😂😂😂.

If We need one of us to go and swear an affidavit, I will go.
@kilemensi and @gertie-sheshe

Reply:
Because:
1. We do not need to use React Context - we may not suffer much over performance related issues, we just don't need it to achieve the validation.
2. We are not using Formik, which eventually uses useFormik anyway as per the docs, so we are not adding extra code.
3. The example is EXACTLY what we have on our code, they would not have it there if it was a wrong or bad approach.

@kilemensi

Reply:
And why can't we use it, if Jared can, even us we can 😩